chain/deals/provider_asks.go

```diff
@@ -143,18 +143,5 @@ func (p *Provider) saveAsk(a *types.SignedStorageAsk) error {
 }
 
 func (c *Client) checkAskSignature(ask *types.SignedStorageAsk) error {
```

Comment:
totally unrelated to your PR: what is this doing in a a file called `provider_asks.go`

Reply:
since it's only one line now I'd just move it to client.go? or client_states.go? is there a client_utils.go? I guess we can reorg once it's in our module.